src/workloads/execution/SlidingWindows.yml

```diff
@@ -0,0 +1,138 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+
+Actors:
+- Name: InsertData
+  Type: Loader
+  Threads: 1
+  Phases:
+  - Repeat: 1
+    Database: &db test
+    Threads: 1
+    CollectionCount: 1
+    DocumentCount: 100000
+    BatchSize: 10000
+    Document:
+      part: {^RandomInt: {min: 1, max: 10}}
+      time: {^RandomDate: {min: 1588626902000, max: 1620162902000}}
+      temp: {^RandomDouble: {distribution: normal, mean: 145, sigma: 30.0}}
+  - &Nop {Nop: true}
+  - *Nop
+
+- Name: Quiesce
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - Repeat: 1
+    Database: admin
+    Operations:
+    - OperationName: RunCommand
+      OperationCommand:
+        fsync: 1
+  - *Nop
+
+- Name: SlidingWindows
+  Type: RunCommand
+  Threads: 1
+  Phases:
+  - *Nop
+  - *Nop
+  - Repeat: 10
+    Database: *db
+    Operations:
+    - OperationMetricsName: MovingAvgPositionBased
+      OperationName: RunCommand
+      OperationCommand:
+          aggregate: Collection0
+          pipeline:
+            [{$setWindowFields: {
+                partitionBy: "$part",
+                sortBy: {time: 1},
+                output: {avg: {$avg: {input: "$temp", documents: [-2, 2]}}}}}]
+          cursor: {batchSize: 10000}
+          allowDiskUse: true
```

Comment:
Was this required/do we actually need to spill for this workload? I ended up filing a different ticket to test this specifically so it may be better to keep this one in-memory

Reply:
When I was testing I noticed some of the runs were failing without this. But that might have been with a very large dataset.

Reply:
I still get memory errors when running without spill to disk.

Reply:
I reduced the number of partitions in order to remove the need for spill.